src/components/shared/Header/Header.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { images, menuItems } from "../../../data/header";
+import "./Header.css";
+
+class Header extends Component {
+  state = {
+    isDisplayed: false,
+    menuItems,
+  };
+
+  /**
+   * Checks the active value to add class "active"
+   * @param {String} active
+   * @param {String} item
+   * @returns {Boolean} If active value equal item name, return "active", else null
+   */
+  isActive = (active, item) => {
```

Comment:
The name convention should be different  
```suggestion
  addActiveClass = (active, item) => {
```